python_client/tests/run_tests.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cd $(dirname $0)
+python3 -m unittest discover --verbose --start-dir tests --top-level-directory=..
```

Comment:
FYI: we use pytest in Fairing. I don't have deep insights on unittest vs pytest but pytest seemed to be a better choice.

Reply:
I found Fairing uses pytest while KFP uses unittest. Unittest seems to be a basic one so I start with it. I will make a switch if pytest makes things way easier.